challenge-06/challenge-06.js

```diff
@@ -58,9 +72,42 @@ Crie uma função chamada `convertToHex`, com as seguintes características:
     a frase:
     "Não temos o equivalente hexadecimal para [COR]."
 */
-// ?
+function convertToHex(color) {
+  
+  var phrase = 'O hexadecimal para a cor ' + color + ' ';
+
+  var white = '#ffffff'; 
```

Comment:
-1
ainda você precisa escrever as coisas duas vezes. Declarar uma variável com a cor, e depois retornar essa variável dentro do case. Para cada nova cor, vai precisar alterar em 2 lugares diferentes.

O ideal é que, sempre que precisar melhorar o seu código, adicionando ou alterando algo, sem mudar a lógica, você possa fazer isso em um único lugar.

O que seria o ideal:
- declarar uma variável sem valor no início da função que receberá a cor hexadecimal;
- no `switch`, você atribui a essa variável a cor conforme o que for passado em `color`;
- no final da função, você retorna a frase, concatenando as duas variáveis :)